embarkConfig/contracts.js

```diff
@@ -113,7 +113,98 @@ module.exports = {
+      // BNT: { instanceOf: "StandardToken" },
+      // MTL: { instanceOf: "StandardToken" },
+      // PPP: { instanceOf: "StandardToken" },
+      // LINK: { instanceOf: "StandardToken" },
+      // KIN: { instanceOf: "StandardToken" },
+      // ANT: { instanceOf: "StandardToken" },
+      // MGO: { instanceOf: "StandardToken" },
+      // MCO: { instanceOf: "StandardToken" },
+      // LRC: { instanceOf: "StandardToken" },
+      // ZSC: { instanceOf: "StandardToken" },
+      // DATA: { instanceOf: "StandardToken" },
+      // RCN: { instanceOf: "StandardToken" },
+      // WINGS: { instanceOf: "StandardToken" },
+      // EDG: { instanceOf: "StandardToken" },
+      // MLN: { instanceOf: "StandardToken" },
+      // MDA: { instanceOf: "StandardToken" },
+      // PLR: { instanceOf: "StandardToken" },
+      // QRL: { instanceOf: "StandardToken" },
+      // MOD: { instanceOf: "StandardToken" },
+      // TAAS: { instanceOf: "StandardToken" },
+      // GRID: { instanceOf: "StandardToken" },
+      // SAN: { instanceOf: "StandardToken" },
+      // SNM: { instanceOf: "StandardToken" },
+      // REQ: { instanceOf: "StandardToken" },
+      // SUB: { instanceOf: "StandardToken" },
+      // MANA: { instanceOf: "StandardToken" },
+      // AST: { instanceOf: "StandardToken" },
+      // R: { instanceOf: "StandardToken" },
+      // 1ST: { instanceOf: "StandardToken" },
+      // CFI: { instanceOf: "StandardToken" },
+      // ENG: { instanceOf: "StandardToken" },
+      // AMB: { instanceOf: "StandardToken" },
+      // XPA: { instanceOf: "StandardToken" },
+      // OTN: { instanceOf: "StandardToken" },
+      // TRST: { instanceOf: "StandardToken" },
+      // TKN: { instanceOf: "StandardToken" },
+      // RHOC: { instanceOf: "StandardToken" },
+      // TGT: { instanceOf: "StandardToken" },
+      // EVX: { instanceOf: "StandardToken" },
+      // ICOS: { instanceOf: "StandardToken" },
+      // DNT: { instanceOf: "StandardToken" },
+      // //"٨": { instanceOf: "StandardToken" },
+      // EDO: { instanceOf: "StandardToken" },
+      // CSNO: { instanceOf: "StandardToken" },
+      // COB: { instanceOf: "StandardToken" },
+      // ENJ: { instanceOf: "StandardToken" },
+      // AVT: { instanceOf: "StandardToken" },
+      // TIME: { instanceOf: "StandardToken" },
+      // CND: { instanceOf: "StandardToken" },
+      // STX: { instanceOf: "StandardToken" },
+      // XAUR: { instanceOf: "StandardToken" },
+      // VIB: { instanceOf: "StandardToken" },
+      // PRG: { instanceOf: "StandardToken" },
+      // DPY: { instanceOf: "StandardToken" },
+      // CDT: { instanceOf: "StandardToken" },
+      // TNT: { instanceOf: "StandardToken" },
+      // DRT: { instanceOf: "StandardToken" },
+      // SPANK: { instanceOf: "StandardToken" },
+      // BRLN: { instanceOf: "StandardToken" },
+      // USDC: { instanceOf: "StandardToken" },
+      // LPT: { instanceOf: "StandardToken" },
+      // ST: { instanceOf: "StandardToken" }
```

Comment:
Can we put this in another file or function. We'll probably need to reuse it too (or maybe put in default)